addon/components/password-field.js

```diff
@@ -23,6 +24,12 @@ export default class PasswordField extends BaseComponent {
   className = 'cs-component-password-field';
   attributesToCopy = [ 'minLength', 'mustIncludeUppercase', 'mustIncludeLowercase', 'mustIncludeNumber', 'mustIncludeSpecialCharacter', 'successMessage'];
 
+  constructor(...args) {
+    super(...args);
+
+    this.value = this.args.value;
```

Comment:
I don't think you need a constructor to do this. You could do
```
@tracked value = this.args.value
```

for something a bit more concise.

Reply:
Wut, that's a thing?? Seems obvious now in hindsight, I've done so many workarounds to deal with this type of thing.

Reply:
I think so. I've used it in a handful of places and it seems to work. I'll double check.